lib/streamlit/bootstrap.py

```diff
@@ -266,18 +267,42 @@ def _maybe_print_old_git_warning(script_path: str) -> None:
         click.secho("  To enable this feature, please update Git.", fg="yellow")
 
 
-def _install_config_watchers():
+def load_config_options(flag_options: Dict[str, Any]):
+    """Load config options from config.toml files, then overlay the ones set by
+    flag_options.
+
+    The "streamlit run" command supports passing Streamlit's config options
+    as flags. This function reads through the config options set via flag,
+    massages them, and passes them to get_config_options() so that they
+    overwrite config option defaults and those loaded from config.toml files.
+
+    Parameters
+    ----------
+    flag_options : Dict[str, Any]
+        A dict of config options where the keys are the CLI flag version of the
+        config option names.
+    """
+    options_from_flags = {
+        opt_name.replace("_", "."): val
+        for opt_name, val in flag_options.items()
```

Comment:
Nit: I would prefer either `name, value` or `option_name, option_value`.

Reply:
Done. Not entirely sure why I chose `opt_name` and `val` originally given they're not consistent with each other